src/main/java/com/robomwm/prettysimpleshop/feature/ShowoffItem.java

```diff
@@ -233,9 +237,37 @@ private void onShopOpen(ShopOpenCloseEvent event)
     private void onShopBreak(ShopBreakEvent event)
     {
         despawnItem(event.getShopInfo().getLocation().add(0.5, 1.2, 0.5));
+        Container container = shopAPI.getContainer(event.getShopInfo().getLocation());
+        InventoryHolder holder = container.getInventory().getHolder();
+        if (!(holder instanceof DoubleChest)) {
+            return;
+        }
+        new BukkitRunnable() {
+            @Override
+            public void run() {
+                spawnItem(new ShopInfo(shopAPI.getLocation(container), shopAPI.getItemStack(container), shopAPI.getPrice(container)));
+            }
+        }.runTaskLater(plugin, 1L);
```

Comment:
Um, isn't the idea to despawn the item, not attempt to spawn another??

Reply:
no

Reply:
Um, then I'm not following why you made this change then.

Reply:
if you just cancel the despawn event (which is what was done before this change) it just creates a new item without the metadata tag

Reply:
Ok, I don't think I'm following what's going on then - I guess I'll have to review the code.

Perhaps this may help me better understand: what did you change, and why did you make this change?